taskt/Core/Automation/Commands/If Commands/BeginMutliIfCommand.cs

```diff
@@ -215,13 +215,13 @@ private void CreateIfCondition(object sender, EventArgs e)
             var automationCommands = taskt.UI.CustomControls.CommandControls.GenerateCommandsandControls().Where(f => f.Command is BeginIfCommand).ToList();
```

Comment:
Could likely be reduced to CommandControls.GenerateCommandsandControls

Reply:
Same is the case here that it's just updating variable names/references in this Command File.